components/widgets/users/avatar.tsx

```diff
@@ -6,9 +6,9 @@ import React from 'react';
 
 import './avatar.scss';
 
-const Avatar = ({url, username, size}) => (
+const Avatar = ({url, username, size = 'md'}: {url: string; username?: string; size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl'}) => (
```

Comment:
The parameter for size can just be `string`.

Reply:
I prefer to specify the options because the list of acceptable options are restrcited.

Reply:
Here I would prefer to use something like:

```
type Props = {
 name: string,
};
const Avatar: React.SFC<Props> = ({url, username, size}) => {
 ...
}
```

And remove the propTypes and the defaultProps

Reply:
:heavy_check_mark: